internal/build/docker.go

```diff
@@ -232,7 +232,10 @@ func (l *localDockerBuilder) startContainer(ctx context.Context, imgRef string,
 	config := &container.Config{Image: imgRef}
 	if cmd != nil {
 		config.Cmd = cmd.Argv
+	} else {
+		config.Cmd = []string{"sh", "-c", "# NOTE(nick): a fake entrypoint"}
```

Comment:
nit: it's not an entrypoint, it's a cmd :P so maybe `a no-op cmd`

also, consider a comment about the fact that an entrypoint takes precedence, for folks unfamiliar with that bit of docker triva

Reply:
will do